### PR TITLE
use shp datatype for mean-per-zone tool

### PR DESCRIPTION
**Pull request by annefou**
Instead of using separate files, we now use proper shapefile datatype.

### CODE REVIEW

**Comment by annefou** [comment]
shp datatype not recognized yet.

**Comment by bgruening** [comment]
Please change https://github.com/NordicESMhub/galaxy-tools/blob/master/.travis.yml#L76 to 19.05

**Comment by annefou** [comment]
Thank you!!!

File: tools/mean-per-zone/mean-per-zone.xml [hunk 1]
```diff
@@ -8,12 +8,9 @@
         <requirement type="package" version="1.4.3.2">netcdf4</requirement>
```
Comment (bgruening) [inline tools/mean-per-zone/mean-per-zone.xml]: please increase the tool version of this tool

**Comment by bgruening** [comment]
Sorry, what I meant is the tool-id.

https://github.com/NordicESMhub/galaxy-tools/pull/33/files#diff-45345fc64dd0546e3c6a5157b657bf38R1

So we get a new versoin in Galaxy. This is important for reproducibility.

**Comment by annefou** [comment]
Ah yes! of course. It makes sense! Sorry, I am just a bit slow!!! 
Thank you for your patience!!!

**Pull request merged**